streamx-common/src/main/scala/com/streamxhub/streamx/common/domain/FlinkVersion.scala

```diff
@@ -50,7 +51,8 @@ class FlinkVersion(val flinkHome: String) extends java.io.Serializable {
 
   lazy val flinkLib: File = {
     require(flinkHome != null, "[StreamX] flinkHome must not be null.")
-    require(new File(flinkHome).exists(), "[StreamX] flinkHome must be exists.")
+    val home = new File(flinkHome)
+    require(home.exists(), "[StreamX] flinkHome must be exists.")
```

Comment:
require(new File(flinkHome).exists(), "[StreamX] flinkHome must be exists.") 

is better?